rke/k8s_rke_system_images.go

```diff
@@ -4543,46 +4543,46 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
-			Flannel:                   m("quay.io/coreos/flannel:v0.13.0-rancher1"),
-			FlannelCNI:                m("rancher/flannel-cni:v0.3.0-rancher6"),
-			CalicoNode:                m("quay.io/calico/node:v3.16.5"),
-			CalicoCNI:                 m("quay.io/calico/cni:v3.16.5"),
-			CalicoControllers:         m("quay.io/calico/kube-controllers:v3.16.5"),
-			CalicoCtl:                 m("quay.io/calico/ctl:v3.16.5"),
-			CalicoFlexVol:             m("quay.io/calico/pod2daemon-flexvol:v3.16.5"),
-			CanalNode:                 m("quay.io/calico/node:v3.16.5"),
-			CanalCNI:                  m("quay.io/calico/cni:v3.16.5"),
-			CanalControllers:          m("quay.io/calico/kube-controllers:v3.16.5"),
-			CanalFlannel:              m("quay.io/coreos/flannel:v0.13.0-rancher1"),
-			CanalFlexVol:              m("quay.io/calico/pod2daemon-flexvol:v3.16.5"),
-			WeaveNode:                 m("weaveworks/weave-kube:2.7.0"),
-			WeaveCNI:                  m("weaveworks/weave-npc:2.7.0"),
-			AciCniDeployContainer:     m("noiro/cnideploy:5.1.1.0.1ae238a"),
-			AciHostContainer:          m("noiro/aci-containers-host:5.1.1.0.1ae238a"),
-			AciOpflexContainer:        m("noiro/opflex:5.1.1.0.1ae238a"),
-			AciMcastContainer:         m("noiro/opflex:5.1.1.0.1ae238a"),
-			AciOpenvSwitchContainer:   m("noiro/openvswitch:5.1.1.0.1ae238a"),
-			AciControllerContainer:    m("noiro/aci-containers-controller:5.1.1.0.1ae238a"),
-			AciGbpServerContainer:     m("noiro/gbp-server:5.1.1.0.1ae238a"),
-			AciOpflexServerContainer:  m("noiro/opflex-server:5.1.1.0.1ae238a"),
-			PodInfraContainer:         m("gcr.io/google_containers/pause:3.2"),
-			Ingress:                   m("rancher/nginx-ingress-controller:nginx-0.35.0-rancher2"),
-			IngressBackend:            m("rancher/nginx-ingress-controller-defaultbackend:1.5-rancher1"),
-			MetricsServer:             m("gcr.io/google_containers/metrics-server:v0.3.6"),
-			CoreDNS:                   m("coredns/coredns:1.7.0"),
-			CoreDNSAutoscaler:         m("rancher/cluster-proportional-autoscaler:1.8.1"),
-			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.6"),
-			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.13"),
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.4.13-rancher1",
+			Kubernetes:                "rancher/hyperkube:v1.19.9-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.74",
+			NginxProxy:                "rancher/rke-tools:v0.1.74",
+			CertDownloader:            "rancher/rke-tools:v0.1.74",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.74",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.15.10",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.15.10",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.15.10",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.1",
+			Flannel:                   "quay.io/coreos/flannel:v0.13.0-rancher1",
```

Comment:
As we are moving away from the non-existent tags or mixed `rancher/` and non existent tags, is this what we are keeping? I would have expected to see this come from `rancher/` as its built by Rancher (vs etcd where we technically only mirror upstream but Flannel is released by us)

Reply:
Was unintentional, changed to `rancher/coreos-flannel:v0.13.0-rancher1`.